feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/group/ClaimGroupViewModel.kt

```diff
@@ -0,0 +1,53 @@
+package com.hedvig.android.odyssey.search.group
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.hedvig.android.odyssey.model.SearchableClaim
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+internal class ClaimGroupViewModel(
+  private val getClaimEntryPoints: GetClaimEntryGroupUseCase,
+  private val groupId: String,
+) : ViewModel() {
+  private val _uiState = MutableStateFlow(ClaimGroupUiState())
+  val uiState = _uiState
+
+  init {
+    loadClaimGroup()
```

Comment:
Doesn't have any way to show error and allow retrying right now right, something we wanna add later?

Reply:
Good point, ill add it